docs/Contributing/Releasing-Fleet.md

```diff
@@ -91,7 +91,11 @@ Note: Please prefix versions with `fleet-v` (e.g., `fleet-v4.0.0`) in git tags,
 > publish a prerelease of fleetctl while the most recent fleetctl npm package, available for public
 > download, is still the latest _official_ release.
 
-5. Announce the release in the #fleet channel of [osquery
+5. Deploy the new version to Fleet's internal dogfood instance: https://fleetdm.com/handbook/engineering#deploying-to-dogfood.
+
+6. In the #g-infrastructure Slack channel, notify the DRI for upgrading Fleet Sandbox that the new version is ready to deploy.
```

Comment:
Makes sense

> I would instead ping @sandbox-oncall in that channel.

@chiiph I updated the instructions to reflect this.

> we've been deploying one version behind what we have released

Got it. My understanding is that when we bump the version for Sandbox, the instances that have already been generated (pre-provisioned) will have the old version. Any new instances will use the latest version. Is this right?

Reply:
> My understanding is that when we bump the version for Sandbox, the instances that have already been generated (pre-provisioned) will have the old version. Any new instances will use the latest version.
> Is this right?

Not exactly, because we take the extra step of refreshing all the preprovisioned sandboxes with the new version as well.